universum/analyzers/mypy.py

```diff
@@ -1,31 +1,29 @@
 import argparse
-import sys
 
 from typing import List
 
 from . import utils
 
 
-def main() -> int:
-    settings = mypy_argument_parser().parse_args()
-
-    cmd = [f"python{settings.version}", '-m', 'mypy', '--ignore-missing-imports']
-    if settings.config_file:
-        cmd.append(f'--config-file={settings.config_file}')
-    cmd.extend(utils.expand_files_argument(settings))
-
-    return utils.report_parsed_outcome(cmd, mypy_output_parser, settings.result_file)
-
-
 def mypy_argument_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description="Mypy analyzer")
     utils.add_files_argument(parser)
     parser.add_argument("--config-file", dest="config_file", type=str, help="Specify a configuration file.")
     utils.add_python_version_argument(parser)
-    utils.add_result_file_argument(parser)
     return parser
 
 
+@utils.sys_exit
+@utils.analyzer(mypy_argument_parser())
```

Comment:
Please consider changing mypy_argument_parser and friends from functions into global variables.
All of them are only called once.

Reply:
I fail to see why that would be a good reason to change functions into globals